convert_json_to_txt.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
```

Comment:
1. try/except the imread call.
2. Put the f.close() inside the loop.
3. Copy the description of the dictionary in the documentation above (not only inside the function)
4. Edit your Readme with this description.

Reply:
Name: VIA-to-YOLOv3